### PR TITLE
Add getWebDir to twig "Plugin" extension

### PR DESCRIPTION
**Pull request by AdrienClairembault**
Add a useful `get_plugin_web_dir` function to the `PluginExtension`.

Usage:
```twig
{% set ajax_url = get_plugin_web_dir('my_plugin') ~ '/ajax/my_url.php' %}
```

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets |

### CODE REVIEW

File: src/Application/View/Extension/PluginExtension.php [hunk 1]
```diff
@@ -85,4 +86,21 @@ public function callPluginHookFunction(string $name, $params = null, bool $retur
...
     }
+
+    /**
+     * Call Plugin::getWebDir() with given params.
+     *
+     * @param string  $plugin
+     * @param bool    $full
+     * @param bool    $use_url_base
+     *
+     * @return mixed|void
```
Comment (cedric-anne) [inline src/Application/View/Extension/PluginExtension.php:97]:
```suggestion
     * @return string|null
```

File: src/Application/View/Extension/PluginExtension.php [hunk 2]
```diff
@@ -85,4 +86,21 @@ public function callPluginHookFunction(string $name, $params = null, bool $retur
...
+     */
+    public function getPluginWebDir(
+        string $plugin,
+        bool $full = true,
+        bool $use_url_base = false
+    ) {
```
Comment (cedric-anne) [inline src/Application/View/Extension/PluginExtension.php:103]:
```suggestion
    ) ?string {
```

File: src/Application/View/Extension/PluginExtension.php [hunk 3]
```diff
@@ -85,4 +86,21 @@ public function callPluginHookFunction(string $name, $params = null, bool $retur
...
+        return Plugin::getWebDir($plugin, $full, $use_url_base);
```
Comment (cedric-anne) [inline src/Application/View/Extension/PluginExtension.php:104]: I quickly read the `Plugin::getWebDir()` and the `Plugin::`Plugin::getWebDir()` ()` code and it appears that it does not handle correctly the value returned for a plugin that is not present on disk.

However, a `null` value should be returned here, if `Plugin::getWebDir()` does not return a string.

File: src/Application/View/Extension/PluginExtension.php [hunk 4]
```diff
@@ -85,4 +86,21 @@ public function callPluginHookFunction(string $name, $params = null, bool $retur
...
+     * @param bool    $full
+     * @param bool    $use_url_base
+     *
+     * @return string|null
+     */
+    public function getPluginWebDir(
+        string $plugin,
+        bool $full = true,
+        bool $use_url_base = false
+    ) ?string {
```
Comment (cedric-anne) [inline src/Application/View/Extension/PluginExtension.php:103]:
```suggestion
    ): ?string {
```

**Review by orthagh — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Review by trasher — APPROVED** [review]

**Pull request merged**